README.md

```diff
@@ -36,16 +68,28 @@ jobs:
     #   run: yarn install
 
     - name: Yarn Install (with cache)
-      uses: kevincobain2000/action-cache-http@v2
+      uses: kevincobain2000/action-cache-http@v3
```

Comment:
Currently there is no v3 tag fot this action.

https://github.com/kevincobain2000/action-cache-http/tags

I assume, after fixing these 2 issue https://github.com/kevincobain2000/action-cache-http/issues/5 and https://github.com/kevincobain2000/action-cache-http/issues/4, a v3 would needs to be pushed?

Reply:
Ref: https://github.com/kevincobain2000/action-cache-http/issues/4

Reply:
Right, I mentioned this in the PR:

> "In the README I bumped the version to "3", assuming there would be a major version bump if this is merged."